store/sqlstore/emoji_store.go

```diff
@@ -66,50 +70,51 @@ func (es SqlEmojiStore) GetByName(ctx context.Context, name string, allowFromCac
 }
 
 func (es SqlEmojiStore) GetMultipleByName(names []string) ([]*model.Emoji, error) {
-	keys, params := MapStringsToQueryParams(names, "Emoji")
+	// Creating (?, ?, ?) len(names) number of times.
+	keys := strings.Join(strings.Fields(strings.Repeat("? ", len(names))), ",")
+	args := makeStringArgs(names)
```

Comment:
Interesting that you only made the args part a function. I was thinking of something returning both keys,args given a `[]string` to avoid doing string manipulations in these methods.

Reply:
I think it's better to use squirrel here and avoid all these helpers altogether. I will open an HW ticket for this.